homeassistant/components/google_tasks/todo.py

```diff
@@ -19,6 +19,8 @@
 from .coordinator import TaskUpdateCoordinator
 from .types import GoogleTasksConfigEntry
 
+# Coordinator is used to centralize the data updates
```

Comment:
Sure, but it also applies to service actions targeted against these entities, so this comment is rather confusion or maybe even incorrect.

Reply:
OK, removed the comment, it was copied from the example in the rule, but as you say it was for a sensor.